### PR TITLE
Update _index.en.md

### PR DESCRIPTION
**Pull request by ninavizz**
It is quite subjective and imho rather offensive to prioritize as "most notably," the difference between design that serves aesthetic needs and design that serves interactive needs. Likewise, a not insignificant bulk of information was absent, that is important—and use of the word "pretty" when we are defining who designers are and aren't, is quite loaded and imho needlessly dismissive. 

This may be overkill, but submitting, anyway. Historically, OSS has been more exclusionary than inclusive; and I'd like to see that change.

Also, entirely open to edit suggestions or outright edits by maintainers. My language skills are rather lacking, and I also did this rather quick!

### CODE REVIEW

**Review by jwflory — APPROVED** [review]
Hi @ninavizz, thanks for your contribution to our design portal! 🙏🏻 Your changes look good and help us in our ongoing journey to making this a more comprehensive resource for designers of all backgrounds. I learned new vocabulary and context while reviewing. From my perspective, this looks good to merge. 👍🏻 

For your context, this resource is still in the early stages of creation. We haven't officially launched it yet. This is being worked on as part of the [Sustain OSS Design & UX Working Group](https://sustainoss.org/working-groups/design-and-ux/), which meets monthly. We are still figuring out our vision and goals for this resource, and as such, we also have not laid out clear contributing guidelines or governance for how to review changes yet. So, I am pinging both @Erioldoesdesign and @RichardLitt to review too and see if they have feedback before merging. Please be patient while we work on figuring out how best to set up processes for reviewing and managing content changes.

That said, I am also curious to know how you discovered this repository? Since we haven't officially launched yet, not much time was spent yet on outreach and promotion, so I am keen to hear how you discovered us! Maybe there is something we can learn there and where we can engage others who might be interested in this project.

File: content/definitions/_index.en.md
Comment (Erioldoesdesign) [inline content/definitions/_index.en.md:19]: I do think a common term that is missing from here that crops up in a lot of historical developer/OSS writing/literature (still mostly academic sadly) is the term 'Usability' and often this is the lens that OSS/Developers understand 'design' by so e.g. 'How usable is my OSS?' and often this is default translated into 'How usable is my OSS (by me and people that look, act and think like me)' .

Including the term here might help OSS contributors/developers feel less intimidated by some of the more subject matter expertise design terms so after we merge this I'll like to add a PR to add this term :)

File: content/definitions/_index.en.md
Comment (Erioldoesdesign) [inline content/definitions/_index.en.md:24]: Particularly love this paragraph!

**Review by Erioldoesdesign — APPROVED** [review]
Absolutely happy to approve - comprehensive and written in an effective and accessible way :)

File: content/definitions/_index.en.md
Comment (ninavizz) [reply content/definitions/_index.en.md:19]: No need for a lengthy explanation, including language that ppl outside of design can identify with, is important! 

It has also not been my own experience, that "usability" is in any way restricted to academia. It has been very in-use in industry, the 18yrs I've been doing this—and it's imho an important term to understand. As are the origins of UX—which are not in academia, but in industry. 

That said, in OSS I have also observed the trend of honoring knowledge (and to an extent, practitioners) from academia, more than knowledge from where it was developed—which was in industry. As someone who's been doing UX for +18yrs, after a career in traditional design, that's been pretty erasive and shitty to experience. Academia is financially un-tenable to most Americans (and many others in the world), as are advanced degrees—and a subtext I've often picked-up in SimplySecure doings, is a preference to overlook practice development from industry folks, in favor of honoring academic developments. 

While America is not the center of the universe (don't tell her that!), there is an entire community of anti-Capitalistic/anti-Colonist UX professionals between here and Europe, hungry to do the kind of work OSS needs UX contributors to work on—and upholding academia as a source-of-truth (or where the most intellectually curious, exist; or where the most ethics-oriented practitioners exist) as often as this community does, has been a big problem I'd like to see confronted. 

Sorry, a bit of a tangent. But, an important thing I'd like to see this community address.

**Comment by ninavizz** [comment]
> That said, I am also curious to know how you discovered this repository? Since we haven't officially launched yet, not much time was spent yet on outreach and promotion, so I am keen to hear how you discovered us! Maybe there is something we can learn there and where we can engage others who might be interested in this project.

I think in the OC or SimplySecure Slack. 

IxDA's slack would be a great place to get more people involved—however, per my above note in response to Eriol's comment, I do feel a reckoning needs to take place in the existing/smol OSS UX community to decide who, exactly, they want to include; what kind of support is to be extended to newcomers, and what is the expectation for supporting that kind of work (bringing newcomers into OSS)— volunteer, or paid. As a veteran UX professional new to OSS, my entry was difficult—and the support I received was inconsistent and at times confusing, while draining more of my resources than adding to them. Trauma in design, also happens amongst design professionals—and that needs to be honored, learned about, and moved away from with intent.

**Comment by ninavizz** [comment]
@RichardLitt Thx for making the appointment—I look forward to chatting!

Until then, to most specifically speak to your latter point: I'm a UX designer, and not at all a user of OSS. I just really believe in OSS. I'm older. I'm American. Between those three, it's not my expectation that for a community as niche and small as UX in OSS, there be multiple affinity groups. SimplySecure, Sustain, OpenSourceDesign. That there is, makes things very confusing. Furthermore, I support my family as the primary income provider—and UX is my job. I don't do this as a hobby, it needs to pay my bills—and preferably working an average of 40hrs a week or fewer. Because, yay (sarcasm), we live in America and that's how life here with that passport and without a spouse with a plush job or a trust fund, works.

My reservations are two fold: One, how might OSS be defining UX, in ways that would not prepare people for careers in industry, or that would (intentionally or not) fail to honor parts of design practice that are important yet less common in academia or non-profit? 

Two, for people interested in taking on OSS projects, there needs to be a broader kind of professionalism and a respect for one's need to get their bills paid, that I have not experienced to date. While those experiences were not contained within the metaphorical walls of the Sustain community, UX is a very tight community w/in industry—and expecting outsiders to see three distinct communities and not "one" amalgamated community, I feel is problematic. Especially when the same people contribute to most of the same 3 communities. I've also tried to resolve the problems I had, directly, and was shut down. By a leader in the Sustain community, even though my experiences happened outside the Sustain community. So, developing a means to invite more from industry into a community that I feel is not "safe," when expectations are more likely to get bills paid doing meaningful work than just to hang out and having fun, I am concerned with. Trust matters a great deal, as does practitioner ethics among other practitioners. This is a business, not a social club. That needs to be honored, more.

**Comment by RichardLitt** [comment]
Thanks, Nina. I don't have the time to respond right now, but I will think on what my response will be. Talk to you tomorrow, in any event.

**Comment by RichardLitt** [comment]
Note: I've thought about it, and I don't want to mix efforts. Nina, I'll respond to the above when we talk. I don't have the space to hold this conversation in two places right now. Looking forward to it.

**Pull request merged**
